python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py

```diff
@@ -420,12 +420,14 @@ def load(self, checkpoint_file):
         args = _transform_config_to_namedtuple(self.model_config)
         self.internal = AutoFormer.load_from_checkpoint(checkpoint_file, configs=args)
 
-    def save(self, checkpoint_file):
+    def save(self, checkpoint_file, tune=False):
         """
         save the forecaster.
 
         :param checkpoint_file: The checkpoint file location you want to load the forecaster.
         """
+        if tune:
```

Comment:
I think there is a `self.use_hpo`, could we just use that member so that we don't need our users to explicitly set `tune`?

Reply:
Good catch! Thanks, done.